nat-lab/bin/core-api.py

```diff
@@ -3,20 +3,38 @@
 import json
 import paho.mqtt.client as mqtt  # type: ignore # pylint: disable=import-error
 from dataclasses import asdict, dataclass
+from enum import Enum
+from http import HTTPStatus
 from http.server import BaseHTTPRequestHandler, HTTPServer
 from itertools import count
-from pprint import pprint
-from typing import Optional
+from typing import Dict, Optional
 from uuid import uuid4
 
+DERP_SERVER = {
```

Comment:
Can [this](https://github.com/NordSecurity/libtelio/blob/326a972824ee7c92f211e05acc085905113081de/nat-lab/tests/config.py#L142) be used here?

Reply:
With current set-up I think no. We cannot do direct import from nat-lab/tests/config.py because we're not copying nat-lab/tests to the containers which runs from /bin. Even if we copy them, python dependencies may be not met for nat-lab/tests in the core-api docker container.
I believe we can address it later on with re-designing mesh_api.py and connecting it to core-api.py

Reply:
ah yeah, makes sense, missed that part. all good then